### ISSUE TITLE
Discord?

### ISSUE THREAD

**Issue by SaviDaWavi** [Brawmario/libsm64-godot#10]
I was wondering if there is a discord for this project since it seems very interesting and I see a lot of potential in it.

**Comment by Brawmario** [comment]
No, there isn't a Discord server for this project. I don't think this would warrant one (and frankly I wouldn't want to run a server, at least not just for this). However feel free to reach out, this GitHub repository should be more than enough to discuss anything related to this project. Who knows, maybe this does get big and I'll consider fostering a community, but for now it's just a side project of mine, open for collaboration.
